releases/latest.releases.yaml

```diff
@@ -65,8 +65,8 @@ repositories:
     version: v1.20.0
   blocktestcore:
     type: git
-    url: https://github.com/robotology/blocktest.git
-    version: v2.3.1
+    url: https://github.com/rbotology/blocktest.git
```

Comment:
There is a typo here, robotology is mispelled

Reply:
Thanks, fixed.